packages/modeling/src/operations/booleans/martinez/index.js

```diff
@@ -0,0 +1,139 @@
+/*
+ * Implementation of the Martinez 2D polygon clipping algorithm
+ * Copyright (c) 2018 Alexander Milevski
+ * https://github.com/w8r/martinez
```

Comment:
@platypii i think you should add a note that this was converted by YOU. this wasn't trivial and some credit should be given. ;)

Reply:
done, thanks :-)